src/repositories/evaluation-repository.ts

```diff
@@ -35,3 +69,25 @@ export const updateStatusByAdministrationId = async (
     },
   })
 }
+
+export const countAllByFilters = async (where: Prisma.evaluationsWhereInput) => {
+  const count = await prisma.evaluations.count({
+    where,
+  })
+
+  return count
+}
+
+export const aggregateSumByFilters = async (
+  _sum: Prisma.EvaluationsSumAggregateInputType,
```

Comment:
we'r not using `_sum` here? 🤔

Reply:
My bad, we should be passing and using it kuys, forgot to remove the _sum properties inside that function. Will update this thanks!

Reply:
🤔

Reply:
Oh, I think it was `aggregateSumById` I changed earlier, I thought it was this one. Will update this as well. My bad!